spec/facility_spec.rb

```diff
@@ -23,4 +34,165 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register_vehicle(cruz)' do
+    it 'registers a vehicle' do
+
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@cruz)).to eq(@facility_1.registered_vehicles)
+    end
+  end
+
+  describe '#register_vehicle(bolt)' do
+    it 'registers (bolt)' do
+
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@bolt)).to eq(@facility_1.registered_vehicles)
+    end
+  end
+
+  describe '#register_vehicle(camaro)' do
+
+    it 'registers (camaro)' do
+      @facility_1.add_service('Vehicle Registration')
+
+      expect(@facility_1.register_vehicle(@camaro)).to eq(@facility_1.registered_vehicles)
+    end
+  end
```

Comment:
These are great tests! You could consider registering 3 vehicles in one example block as well, that way you could test that `@facility_1.registered_vehicles` contains all of them.